include/networkit/distance/BidirectionalBFS.hpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * BidirectionalBFS.hpp
+ *
+ *  Created on: 14.06.2019
+ *      Author: Eugenio Angriman <angrimae@hu-berlin.de>
+ */
+
+#ifndef BIDIRECTIONAL_BFS_H
+#define BIDIRECTIONAL_BFS_H
+
+#include <networkit/distance/STSP.hpp>
+
+namespace NetworKit {
+
+/*
+ * @ingroup distance
+ * Implements a bidirectional breadth-first search on a graph from two given source and target
+ * nodes. Explores the graph from both the source and target nodes until the two explorations meet.
+ */
+class BidirectionalBFS final : public STSP {
+
+public:
+    using STSP::STSP;
```

Comment:
Why is this `using` declaration necessary?

Reply:
To use the base class constructor

Reply:
Alright. IMHO this is easy to overlook. We should make the intent explicit with a comment `// Inherit the constructors of STSP.`.

Reply:
Sure I can add it